partcad-cli/src/partcad_cli/click/command.py

```
@@ -3,6 +3,8 @@
 import partcad as pc
 import coloredlogs
 import logging
+import sentry_sdk
+from sentry_sdk.tracing import Transaction
```

Ruff (0.8.2)

`sentry_sdk.tracing.Transaction` imported but unused

Remove unused import: `sentry_sdk.tracing.Transaction`

(F401)